lib/_debugger.js

```diff
@@ -829,13 +829,7 @@ function Interface(stdin, stdout, args) {
   // Run script automatically
   this.pause();
 
-  // XXX Need to figure out why we need this delay
-  setTimeout(function() {
-
-    self.run(function() {
-      self.resume();
-    });
-  }, 10);
+  setImmediate(() => { this.run(() => { this.resume; }); });
```

Comment:
Heh, looks like we don't have a test that covers that. Guess I'll be adding one.